src/test/java/seedu/address/logic/commands/AddApptCommandTest.java

```diff
@@ -205,11 +205,8 @@ public boolean hasPerson(Person person) {
         public void updatePerson(Person personToUpdate, Person updatedPerson) {
             requireAllNonNull(personToUpdate, updatedPerson);
             if (!personToUpdate.isSamePerson(updatedPerson)) {
-                // TODO: what should be an appropriate response?
```

Comment:
You can use assert if the person to be updated is the same person, rather than using the if block.

Reply:
Ok!